phabricator.py

```diff
@@ -61,7 +80,8 @@ def run(self):
 
         # Run `arc browse` and dump the output to the console
         browse_path = '{0}:{1}'.format(filename, lines)
-        arc_args = [settings.get('arc_path', 'arc'), 'browse', browse_path, '--branch', escaped_branch]
+        arc_args = [
+            settings.get('arc_path', 'arc'), 'browse', browse_path, '--branch', escaped_branch]
```

Comment:
It looks like there are a few style specific changes. In the future, it would be nice to handle those via separate PRs to reduce noise.